### PR TITLE
clj: add page

### PR DESCRIPTION
**Pull request by diegobfernandez**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/clj.md
Comment (sbrl) [inline pages/common/clj.md:9]: Wow, this is a pretty complex example to be starting a page with. Are you aware of our `{{token_syntax}}` we've got documented in our [style guide](https://github.com/tldr-pages/tldr/blob/master/contributing-guides/style-guide.md) at all? The page for `git` is a good example of how it's used.

The same goes for the rest of the page.

Comment (diegobfernandez) [reply pages/common/clj.md:9]: oh, my bad. I haven't read that page.
I will change the example to just start a repl without options. that first one is the first example of `clj -h` but I agree it's confusing.

```suggestion
`clj`
```

File: pages/common/clj.md
Comment (sbrl) [inline pages/common/clj.md:11]: Perhaps this will read better?
```suggestion
- Execute a function:
```

File: pages/common/clj.md
Comment (sbrl) [inline pages/common/clj.md:13]: Generally, we prefer to avoid shortening names to aid clarity :-)
```suggestion
`clj -X {{namespace/function_name}}`
```

Also, is there a long form of the argument there? Generally we prefer the long forms of arguments in tldr pages. The page goes for other examples.

File: pages/common/clj.md
Comment (sbrl) [inline pages/common/clj.md:15]: Perhaps this will read better?
```suggestion
- Run the main function of a specified namespace:
```

File: pages/common/clj.md
Comment (sbrl) [inline pages/common/clj.md:19]: Perhaps this will read better?
```suggestion
- Prepare a project by resolving dependencies, downloading libraries, and making / caching classpaths:
```

File: pages/common/clj.md
Comment (sbrl) [inline pages/common/clj.md:25]: What's the difference between this example and the earlier REPL one? This example looks rather long and complicated. Is there a way we could shorten it at all?

File: pages/common/clj.md
Comment (sbrl) [inline pages/common/clj.md:27]: Perhaps this will read better?
```suggestion
- Start REPL for ClojureScript and open a web browser:
```

File: pages/common/clj.md
Comment (diegobfernandez) [reply pages/common/clj.md:25]: This one enables external tools to communicate with the REPL through the nREPL API. The way to make it shorter would be removing CIDER middleware which is nice to have but totally optional for most IDEs/tools.
```suggestion
- Start nREPL server to be used with IDEs/tools:

`clj -Sdeps '{:deps {nrepl/nrepl {:mvn/version "0.7.0"}}}' -m nrepl.cmdline --interactive`
```

Comment (sbrl) [reply pages/common/clj.md:25]: Ah, I see. Thanks for the explanation! Let's keep it as-is then.

File: pages/common/clj.md
Comment (sbrl) [inline pages/common/clj.md:23]: Perhaps this will read better?
```suggestion
- Start an nREPL server with the CIDER middleware:
```

File: pages/common/clj.md
Comment (sbrl) [inline pages/common/clj.md:27]: Perhaps this will read better?
```suggestion
- Start a REPL for ClojureScript and open a web browser:
```

**Review by sbrl — COMMENTED** [review]
Last round of changes, promise! Thanks for sticking with the review process. I know it can feel tedious at times.

**Comment by diegobfernandez** [comment]
> Last round of changes, promise! Thanks for sticking with the review process. I know it can feel tedious at times.

😄 no worries, I am not the best at writing so it's always good to receive feedback I can use to improve.

**Comment by diegobfernandez** [comment]
> Last round of changes, promise! Thanks for sticking with the review process. I know it can feel tedious at times.

😄 no worries, I am not the best at writing so it's always good to receive feedback I can use to improve.

**Review by sbrl — APPROVED** [review]
Thanks, @diegobfernandez!

Once we get a 2nd approval, we can get to merging.

**Review by sahilister — APPROVED** [review]
Thanks @diegobfernandez for contribution and sorry for the wait. 

Welcome aboard!

**Pull request merged**
